src/validation.cpp

```diff
@@ -2196,27 +2035,31 @@ bool static FlushStateToDisk(const CChainParams& chainparams, CValidationState &
     }
     if (full_flush_completed) {
         // Update best block in wallet (so we can detect restored wallets).
-        GetMainSignals().ChainStateFlushed(::ChainActive().GetLocator());
+        GetMainSignals().ChainStateFlushed(m_chain.GetLocator());
     }
     } catch (const std::runtime_error& e) {
         return AbortNode(state, std::string("System error while flushing: ") + e.what());
     }
     return true;
 }
 
-void FlushStateToDisk() {
+void CChainState::ForceFlushStateToDisk() {
     CValidationState state;
     const CChainParams& chainparams = Params();
-    if (!FlushStateToDisk(chainparams, state, FlushStateMode::ALWAYS)) {
+    if (!this->FlushStateToDisk(chainparams, state, FlushStateMode::ALWAYS)) {
```

Comment:
in commit 7ab9e339a8 refactoring: FlushStateToDisk -> CChainState


style-question: You remove the global with the same name in this commit, so the `this->` is not required?

Reply:
I like prefixing with `this->` for methods because it makes it obvious we're calling a method, sort of like `m_`. See the IRC discussion: http://www.erisian.com.au/bitcoin-core-dev/log-2019-05-06.html#l-572

Happy to undo it if it's controversial though.

Reply:
Asked because we don't do this elsewhere. You can pick whatever you prefer ofc

Reply:
> I like prefixing with this-> for methods because it makes it obvious we're calling a method, sort of like m_

FWIW, I still think https://github.com/bitcoin/bitcoin/pull/14635 is a good solution for this that would improve readability in these cases (https://github.com/bitcoin/bitcoin/pull/15976#issuecomment-490557315, https://github.com/bitcoin/bitcoin/pull/15976#issuecomment-490559432, https://github.com/bitcoin/bitcoin/pull/15976#discussion_r284264160)